aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -215,6 +215,20 @@ module aptos_framework::fungible_asset {
         frozen: bool,
     }
 
+    #[event]
+    /// Emitted when fungible assets are minted.
+    struct Mint has drop, store {
+        metadata: address,
+        amount: u64,
+    }
+
+    #[event]
+    /// Emitted when fungible assets are burned.
+    struct Burn has drop, store {
```

Comment:
we don't log if the entire Store object is burned right? should we include where the burn comes from? the object address of Store?

Reply:
also need object address and owner and whether it's primary or secondary store

Reply:
Yes we need to include object address of the store.

Reply:
Synced with @rtso , this `Burn` event is sufficient as there is no way to know which store the FAs are being burned from this function: `public fun burn(ref: &BurnRef, fa: FungibleAsset)`.